docs/architecture/overview.md

```diff
@@ -37,6 +45,25 @@ This document specifies the architecture for GCP Ingestion as a whole.
     - Expected initial topics are Structured Ingestion, Telemetry, and Pioneer
 - Must accept configuration defining HTTP headers to capture
 
+### Account Ecosystem Decryptor
+
+- Must have access restricted to a limited set of operators
+- Must have access to a set of encrypted private keys stored in
+  the source repository or on GCS
+- Must have access to KMS to decrypt the private keys on startup,
+  holding them in memory
+- Must throw away any payload that does not match expected structure
+  - The payload must contain valid JSON
+  - The payload may be compressed via `gzip`
+  - The payload must contain a top-level field called `ecosystem_anon_id`
```

Comment:
This can be true for desktop telemetry pings and for pings sent from FxA, but probably isn't possible from Glean. We can modify this to accommodate a good place for Glean to record these values.

Reply:
It sounds like we'll need to have a set of schemas with the anon_id in different places: desktop, fxa, and glean.
I will also relax this such that anon ID is not required to be present.